robolectric-shadows/shadows-core/src/main/java/org/robolectric/fakes/RoboMenu.java

```diff
@@ -21,7 +23,7 @@
   private Context context;
 
   public RoboMenu() {
-    this(null);
+    this(RuntimeEnvironment.application);
```

Comment:
The thing is...not having an explicit constructor in a subclass like this is exactly how we ended up in this position. I think this is more explicit and clear than what was here prior, given the NPE that I discovered due to lack of clarity.